app/views/layouts/printer_friendly.html.haml

```diff
@@ -5,7 +5,9 @@
       =h html_title
     %style
       a.anchor, a.shy {display:none;}
+      @media print { .screen-only {display: none}}
   %body{class:language_direction}
+    %button.screen-only{:onclick => 'window.history.back()',}back
```

Comment:
hi @ben-pr-p.
thanks for the changes. Could you replace the 'back' with
```
= :back.t
```
so it will be translated?

Thanks!